### PR TITLE
🏌️‍♂️ Golf/put put

### PR DESCRIPTION
**Pull request by cristianbote**

### CODE REVIEW

File: src/core/parse.js [hunk 1]
```diff
@@ -44,7 +42,7 @@ export const parse = (obj, paren, wrapper) => {
             // Push the line for this property
             else {
                 current +=
-                    (prefixer && prefixer(key.replace(/[A-Z]/g, '-$&').toLowerCase(), val)) ||
+                    (parse.p && parse.p(key.replace(/[A-Z]/g, '-$&').toLowerCase(), val)) ||
```
Comment (cristianbote) [inline src/core/parse.js]: @JoviDeCroock what do you think? This would shave around `5B` I think

File: src/styled.js [hunk 2]
```diff
@@ -14,15 +18,23 @@ const setup = (pragma, prefix, forward, theme) => {
...
+
+            // Set a flag if the current components had a previous className
+            // similar to goober. This is the append/prepend flag
             _ctx.o = /\s*go[0-9]+/g.test(_ctx.p.className);
+
+            // Define the new className
             return h(
                 tag,
-                Object.assign({}, props, {
+                Object.assign(_ctx.p, {
```
Comment (JoviDeCroock) [inline src/styled.js]: Ctx.p potentially has a theme in it so we would switch the props reference. This doesn’t seem ideal/deterministic

Comment (cristianbote) [reply src/styled.js]: That's a good catch. Thank you! Size is the same 😃

**Comment by cristianbote** [comment]
Huh, this one doesn't really shaves bytes... 😢 gonna look into it

**Pull request merged**